templates/web/fixmystreet-uk-councils/contact/who.html

```diff
@@ -25,9 +25,15 @@ <h4>Topic:</h4>
 
   <div class="form-error__box form-error--council[% IF dest != 'council' %] hidden[% END %]">
     <p>
+    [% IF c.cobrand.moniker == 'surrey' %]
+    <strong>If your problem is not a street issue</strong>, or is <strong>not suitable for publication on the site</strong>,
```

Comment:
This shows the paragraph twice, it's replacing the first rather than the second paragraph?